proto/Networking/Requests/Social/DeleteGiftFromInventoryMessage.proto

```diff
@@ -0,0 +1,6 @@
+syntax = "proto3";
+package POGOProtos.Networking.Requests.Social;
```

Comment:
Not a social request

Reply:
true

Reply:
fixed https://github.com/pogosandbox/node-pogo-protos/pull/19/commits/2699be05eaf2516680c0121818fc42aa5a287daf#diff-275cf9ff761dcc1b02f825cd21da5af7R2